python/adbc_driver_manager/adbc_driver_manager/dbapi.py

```diff
@@ -195,11 +196,26 @@ def connect(
     Parameters
     ----------
     driver
-        The driver name. For example, "adbc_driver_sqlite" will
-        attempt to load libadbc_driver_sqlite.so on Linux systems,
-        libadbc_driver_sqlite.dylib on MacOS, and
-        adbc_driver_sqlite.dll on Windows. This may also be a path to
-        the library to load.
+        The driver to use.  This can be one of several values:
+
+        - A driver name or manifest name.
+
+          For example, "adbc_driver_sqlite" will first attempt to load
+          adbc_driver_sqlite.toml from the various search paths.  Then, it
+          will try to load libadbc_driver_sqlite.so on Linux,
+          libadbc_driver_sqlite.dylib on MacOS, or adbc_driver_sqlite.dll on
+          Windows.  See :doc:`/format/driver_manifests`.
+
+        - A full path to a shared library to load.
+
+        - Only a URI, in which case the URI scheme will be assumed to be the
+          driver name and will be loaded as above.  This will happen when
+          "://" is detected in the driver name.  (It is not assumed that the
+          URI is actually a valid URI.)
```

Comment:
Maybe mention here that this only works if the driver natively supports URIs with a scheme that happens to be the same as the driver name (since the driver manager does _not_ strip off the scheme before passing it to the driver).